### PR TITLE
Sprint 13 -> dev

### PR DESCRIPTION
**Pull request by mjoyce91**
Work completed during Sprint 13. Includes:

✨ Sprint Tasks

- https://github.com/18F/State-TalentMAP/pull/1270 - CDO Portfolio "No handshake" filter tab
- https://github.com/18F/State-TalentMAP/pull/1271 - Home page card redesign
- https://github.com/18F/State-TalentMAP/pull/1279 - Comparison page redesign
- https://github.com/18F/State-TalentMAP/pull/1280 - Bidder portfolio additional details (waivers, bids, assignment)
- https://github.com/18F/State-TalentMAP/pull/1284 - Tour End Date -> Estimated End Date (wording)
- https://github.com/18F/State-TalentMAP/pull/1286 - Add bid cycle to search results cards
- https://github.com/18F/State-TalentMAP/pull/1301 - Draft bid submission and deletion from Bid Tracker

🐛 Bug fixes & tweaks

- https://github.com/18F/State-TalentMAP/pull/1251 - update run time dependencies
- https://github.com/18F/State-TalentMAP/pull/1255 - Sass/JS variable sharing
- https://github.com/18F/State-TalentMAP/pull/1256 - AuthorizedWrapper
- https://github.com/18F/State-TalentMAP/pull/1257 - update dev dependencies
- https://github.com/18F/State-TalentMAP/pull/1258 - resolve `setState` warnings with the Glossary
- https://github.com/18F/State-TalentMAP/pull/1259 - Add `preventDefault()` to a button to avoid query params being added to url
- https://github.com/18F/State-TalentMAP/pull/1261 - Only fetch notifications if token exists
- https://github.com/18F/State-TalentMAP/pull/1292 - Import dynamic `$asset-path` to .scss files
- https://github.com/18F/State-TalentMAP/pull/1298 - Fix IE11 button styles
- https://github.com/18F/State-TalentMAP/pull/1283 - Checkbox wording/style changes based on client feedback

🖥️ Accessibility

- https://github.com/18F/State-TalentMAP/pull/1262 - gray -> black placeholder text
- https://github.com/18F/State-TalentMAP/pull/1294 - adds titles to profile menu buttons

### CODE REVIEW

File: src/Components/MediaQuery/MediaQuery.jsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import MediaQuery from 'react-responsive';
-import breakpoints from '../../Constants/breakpoints';
```
Comment (burgwyn) [inline src/Components/MediaQuery/MediaQuery.jsx]: Can we delete this file?

File: src/Components/SaveNewSearchContainer/SaveNewSearchContainer.jsx [hunk 2]
```diff
@@ -16,7 +16,9 @@ class SaveNewSearchContainer extends Component {
     };
   }
 
-  toggleInput() {
+  toggleInput(e) {
+    // preventDefault() to avoid query parmas getting added in MS Edge
```
Comment (burgwyn) [inline src/Components/SaveNewSearchContainer/SaveNewSearchContainer.jsx]: `params`

File: src/Components/BidderPortfolio/BidderPortfolioGridItemAdditional/ItemList/ItemList.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ItemListing from './ItemListing';
+
+const BidderPortfolioGridItemAdditional = ({ title, items }) => (
```
[the remark this answers is not in the document]
Comment (mjoyce91) [reply src/Components/BidderPortfolio/BidderPortfolioGridItemAdditional/ItemList/ItemList.jsx]: There's an `ItemList` component that's a container for the `ItemListing` component and a title prop.

File: src/Components/MediaQuery/MediaQuery.jsx (hunk 1 above)
Comment (mjoyce91) [reply src/Components/MediaQuery/MediaQuery.jsx]: Looks like it already is 👍

File: src/Components/SaveNewSearchContainer/SaveNewSearchContainer.jsx (hunk 2 above)
Comment (mjoyce91) [reply src/Components/SaveNewSearchContainer/SaveNewSearchContainer.jsx]: 🧀

File: src/Components/BidderPortfolio/BidderPortfolioGridItemAdditional/ItemList/ItemList.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (mjoyce91) [reply src/Components/BidderPortfolio/BidderPortfolioGridItemAdditional/ItemList/ItemList.jsx]: Changed so that component name and file name match.

**Pull request merged**